frappe/public/js/frappe/form/controls/phone.js

```diff
@@ -22,10 +35,10 @@ frappe.ui.form.ControlPhone = class ControlPhone extends frappe.ui.form.ControlD
 		// Replaces code when selected and removes previously selected.
 		this.country_code_picker.on_change = (country) => {
 			if (!country) {
-				return this.reset_inputx();
+				return this.reset_input();
```

Comment:
Already fixed this line in PR https://github.com/frappe/frappe/pull/17426 😅

Reply:
That's fine, git will merge it anyway :p